### PR TITLE
copy recs, config options, and overview

### PR DESCRIPTION
**Pull request by MiaAltieri**
this is literally just copying over already reviewed text and adding the word "Kubernetes" to it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,29 @@
-# Charm for MongoDB on Kubernetes
+# Charmed MongoDB on Kubernetes
+## Overview
 
-## Description
+The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database. It is an open source, end-to-end, production ready data platform on top of cloud native technologies.
```
Comment (WRFitch) [inline README.md:4]: you may want to clarify what "day 0 to day 2" means, or link to a clarification - as written, it implies that you're on your own after three days.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,29 @@
...
 
-Fully supported solution for production-grade MongoDB on Kubernetes from Canonical.
+MongoDB is a popular NoSQL database application. It stores its data with JSON-like documents creating a flexible experience for users; with easy to use data aggregation for data analytics. It is a distributed database, so vertical and horizontal scaling come naturally.
```
Comment (WRFitch) [inline README.md:6]: Again, you could link to MongoDB documentation here

**Review by WRFitch — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (MiaAltieri) [reply README.md:4]: I also got tripped up on this awhile ago. Not the biggest fan, but it is a phrase that you'll see all over Canonicals docs -- and all over the software world. 

It is defined as "Day 0, Day 1, and Day 2 refer to different phases of the software life cycle. Day 0 is the design phase, Day 1 is where things get creative, and Day 2 is just a daily operations routine."

File: README.md [hunk 3]
```diff
@@ -1,16 +1,29 @@
...
 
-Run the developer's favourite document database - MongoDB! Charm for MongoDB is a
-fully supported, automated solution from Canonical for running production-grade
-MongoDB on Kubernetes. It offers simple, secure and highly available setup with
-automatic recovery on failover. The solution includes scaling and other capabilities.
+This operator charm deploys and operates MongoDB on Kubernets. It offers features such as replication, TLS, password rotation, and easy to use integration with applications. The Charmed MongoDB Operator meets the need of deploying MongoDB in a structured and consistent manner while allowing the user flexibility in configuration. It simplifies deployment, scaling, configuration and management of MongoDB in production at scale in a reliable way.
+
+## Requirements 
+- 2GB of RAM.
```
Comment (Mehdi-Bendriss) [inline README.md:11]: Would it be worth adding `at least` before "2GB of RAM" and "2 CPU.."? or is that implied already?

**Review by Mehdi-Bendriss — DISMISSED** [review]
LGTM! with a few nits

**Review by zmraul — APPROVED** [review]
Looking good!

File: README.md (hunk 1 above)
Comment (MiaAltieri) [reply README.md:4]: I would but the link but I am worried about the link changing over time . There is no wiki for this defintion so the other links might change over time

File: README.md (hunk 2 above)
Comment (MiaAltieri) [reply README.md:6]: Since we already have one link to MongoDB I will refrain here

**Review by zmraul — APPROVED** [review]

**Review by Mehdi-Bendriss — APPROVED** [review]

**Review by WRFitch — APPROVED** [review]

**Review by deusebio — APPROVED** [review]
All good!

**Pull request merged**